docarray/documents/text.py

```diff
@@ -102,10 +104,25 @@ class MultiModalDoc(BaseDoc):
 
     """
 
-    text: Optional[str]
-    url: Optional[TextUrl]
-    embedding: Optional[AnyEmbedding]
-    bytes_: Optional[bytes]
+    text: Optional[str] = Field(
+        description='The text content stored in the document',
+        example='This is an example text content of the document',
+    )
+    url: Optional[TextUrl] = Field(
+        description='''The url of the text content. When text content is too long
+    to be stored inline or in a file, the remote url can be used to load the
+        text content''',
+        example='https://www.w3.org/History/19921103-hypertext/hypertext/README.html',
+    )
+    embedding: Optional[AnyEmbedding] = Field(
+        description='''Embedding field is used to store tensor objects of type
+        Tensorflow, PyTorch, and NumPy''',
+        example='''np.zeros((3, 32, 32))''',
```

Comment:
I think the example would be best as a list, so that is shown properly in the `SwaggerUI` when exposed via FastAPI.

Reply:
agree with @JoanFM I would even remove the mention of Numpy and Pytorch, this description is more useful for Swagger UI

Reply:
so the mention of tensor objects should only be `Tensorflow` and `JAX`

Reply:
You can keep all or remove all, but not having partial list. I believe @samsja says we do not need to specify these types. As for SwaggerUI this is not so helpful

Reply:
got it so the new description would be `Embedding field is used to store tensor objects`? or should it be something else?